### PR TITLE
Fix compilation in Xcode 10.x

### LINKED ISSUE #133: [xcode 10] Build crashes
# Bug

Build crashes on IOS, because [labelColor variable](https://github.com/react-native-community/datetimepicker/blob/66be0879c324a69d250d5e6067b0f69683901f80/ios/RNDateTimePickerManager.m#L51) does not exist.

Actual error message:
```
.../node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerManager.m:51:33: 
error: no known class method for selector 'labelColor'
        defaultColor = [UIColor labelColor];
```

## Environment info

Library version: 2.3.0
IOS: 13.2.2

### PR DESCRIPTION
**Pull request by luancurti**
# Summary

Fix compilation in Xcode 10.x

Fix #133 

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ✅     |

## Checklist

- [X] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

File: ios/RNDateTimePickerManager.m [hunk 1]
```diff
@@ -48,7 +48,9 @@ - (UIView *)view
   } else {
     UIColor* defaultColor;
     if (@available(iOS 13.0, *)) {
+#ifdef __IPHONE_13_0
```
Comment (vonovak) [inline ios/RNDateTimePickerManager.m:51]: did you test this with xcode 10? Shouldn't the `ifdef` wrap the entire if? like here https://stackoverflow.com/a/58083853/2070942

Comment (luancurti) [reply ios/RNDateTimePickerManager.m:51]: No I'm with Xcode 11, but I think you are right, I'll download Xcode 10 and test this, thanks

Comment (vonovak) [reply ios/RNDateTimePickerManager.m:51]: I don't think you need to go through that hassle, I was just asking. Frankly, I don't think we're required to support xcode 10, it's already 6 months old. I think if someone needs that support, they should contribute it, we should not be spending hours by downloading xcode 10...

Comment (luancurti) [reply ios/RNDateTimePickerManager.m:51]: I agree. So we need to update the readme requiring use of Xcode 11, what do you think?

Comment (vonovak) [reply ios/RNDateTimePickerManager.m:51]: yes, that sounds good. :)

Comment (Maickonn) [reply ios/RNDateTimePickerManager.m:51]: Hello, @vonovak , I tested on XCode 10 and worked fine.

**Pull request closed without merging**